src/android/ChromeCustomTabPlugin.java

```diff
@@ -43,6 +44,7 @@ public void initialize(CordovaInterface cordova, CordovaWebView webView) {
 
     @Override
     public boolean execute(String action, JSONArray args, CallbackContext callbackContext) throws JSONException {
+        final Context context = this.cordova.getContext();
```

Comment:
is this different than the `callbackContext` argument?

Reply:
Yes, it is. See [`android.content.Context`](https://developer.android.com/reference/android/content/Context) and [`org.apache.cordova.CallbackContext`](https://github.com/apache/cordova-android/blob/master/framework/src/org/apache/cordova/CallbackContext.java)